src/components/Practice/AudioTooltipModal.jsx

```
@@ -61,58 +62,67 @@ const AudioTooltipModal = ({ audioSrc, description, children }) => {
+          background: "#fff",
+          border: "2px solid #ff8c52",
+          borderRadius: "16px",
+          boxShadow: "0px 0px 20px 2px #FF7F367A",
+          padding: "10px 0px",
+          marginTop: "8px",
+          zIndex: 1000,
+          textAlign: "center",
+          width: "180px",
+          height: "50px",
+        }}
+      >
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)